PREREQUISITES.md

```diff
@@ -254,6 +254,14 @@ aws-vault --debug login ${ACCOUNT_NAME}-ro
 
 ```
 
+For future convenience, add this to your .bashrc:
```

Comment:
This section is potentially misleading for folks, like me, who are using multiple accounts on a daily basis.

I'd probably suggest something like:

```
alias vault-exec-${ACCOUNT_NAME} ...
```
Or something similar.